docs/walkthrough/walkthrough-logs.md

```diff
@@ -151,7 +153,7 @@ kind: ClusterFlow
 metadata:
   name: flow
 spec:
-  outputRefs:
+  globalOutputRefs:
     - s3
   selectors:
```

Comment:
This is deprecated, so better to use the match / select syntax.

Reply:
Fixed, thanks.